File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.py
while(True):
  num = input()
  index = 0
  if num == '0':
    break;

  for i in range(len(num) // 2):
    if num[i] != num[len(num) - 1 - i]:
      print("no")
      break;
    else:
      index += 1

  if index == len(num) // 2:
    print("yes")

File: 백준/Bronze/1259. 팰린드롬수/팰린드롬수.c
#include <stdio.h>
#include <string.h>

int main(void) {  
   char num[6];
   int i;

   while(1){
      scanf("%s", num);

      if(num[0] == '0') break;

      for(i = 0; i < strlen(num) / 2; i++){
         if(num[i] != num[strlen(num) - 1 - i]){
            printf("no\n");
            break;
         }
      }
      if(i == strlen(num) / 2) printf("yes\n");
   }
   return 0;
}